src/components/Cart/Cart.js

```diff
@@ -7,15 +8,29 @@ import {
   BottomRow,
   ClearCart,
 } from "./styled";
+import CartModal from "../Modals/CartModal";
+import { GlobalStyle } from "../Modals/styled";
```

Comment:
still it should not be here. It should be in highest level of styling, like index.js or App.js